modules/data-entry/src/main/frontend/src/questionnaire/Subject.jsx

```diff
@@ -503,13 +503,13 @@ function FormData(props) {
     const questionTitle = entryDefinition["text"];
 
     if (typeof(existingQuestionAnswer?.[1]?.value) != "undefined" && (displayed < maxDisplayed)) {
-      let existingAnswerValue = existingQuestionAnswer[1]["value"];
-      // TODO: Other question types will need to be handled as well
+      let existingAnswerValue = existingQuestionAnswer[1]["displayedValue"];
```

Comment:
This should probably be renamed now to make it clear that we're working with the displayed value, not with the true value, and that it's an array. I suggest the term "pretty print" since it's a commonly used to specify "making things easier to read".
```suggestion
      let prettyPrintedAnswers = existingQuestionAnswer[1]["displayedValue"];
```